src/feature-viewer.js

```diff
@@ -1554,8 +1554,10 @@ var FeatureViewer = (function () {
                 .attr("in", "SourceGraphic");
 
             svgContainer.on('mousemove', function () {
-                var absoluteMousePos = SVGOptions.brushActive ? d3.mouse(d3.select(".background").node()) : d3.mouse(svgContainer.node());; 
-                $(div + " #zoomPosition").text(Math.round(scalingPosition(absoluteMousePos[0])));
+                var absoluteMousePos = SVGOptions.brushActive ? d3.mouse(d3.select(".background").node()) : d3.mouse(svgContainer.node());;          
+                var pos = Math.round(scalingPosition(absoluteMousePos[0]));
+                pos += sequence[pos] || "";
```

Comment:
Shouldn't it be ```pos += sequence[pos-1] || ""; ``` ?
the scale range begins at 1, but the sequence first index is 0.